kitsune/users/management/commands/clear_expired_auth_tokens.py

```diff
@@ -1,11 +0,0 @@
-from datetime import datetime, timedelta
```

Comment:
We still have a reference to this in scripts/cron.py#job_clear_expired_auth_tokens

We should remove the DMS_CLEAR_EXPIRED_AUTH_TOKENS setting from settings.py and our k8s files, and remember to remove it from DMS when we deploy to prod.

Do we also need a migration to nuke all Token objects from the DB?

Reply:
Probably we do but I wouldn't do it at this point. We are closing down all the paths to use these tokens. After the release we can clean everything up